spec/rails_helper.rb

```diff
@@ -50,5 +50,10 @@
     driven_by(:rack_test)
   end
 
+  config.before(:each, type: :feature) do
+    create(:academic_cycle, :current)
+    create(:academic_cycle, next_cycle: true)
```

Comment:
In the end-to-end tests, the trainees are created in the next academic cycle. This academic cycle needs to be present in order for the funding section to work.